docker/dev/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:carbon
+
+LABEL AUTHOR="Crispus Kamau <crispus.kamau@andela.com>"
+LABEL application="soc-frontend"
+
+#due to the rapid changes the serve team is making please 
+#use this version of serve
+
+RUN yarn global add express@4.16.2 morgan@1.9.1
+
+WORKDIR /application
+
+COPY public /application
+
+COPY sandserver /application
+
+ENV NODE_PATH /usr/local/share/.config/yarn/global/node_modules/
```

Comment:
this could move further up the initial layers

Reply:
I have fixed this.